### PR TITLE
Update docs with API changes

### PR DESCRIPTION
**Pull request by vstefanovic97**
I just realised I missed updating the readme in https://github.com/embroider-build/content-tag/pull/62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,19 @@ let p = new Preprocessor();
 let output = p.parse('<template>Hi</template>');
 ```
 
+#### `PreprocessorOptions`
+
+````ts
+interface PreprocessorOptions {
+
+  /** Default is `false` */
+  inline_source_map: boolean;
```
Comment (NullVoxPopuli) [inline README.md:80]:
```suggestion
  inline_source_map?: boolean;
```

**Review by mansona — APPROVED** [review]
Looks good 🎉

File: README.md [hunk 2]
```diff
@@ -71,6 +71,19 @@ let p = new Preprocessor();
...
+
+  filename: string;
```
Comment (NullVoxPopuli) [inline README.md:82]: Is this one optional?

Should be ?: as well?

File: README.md (hunk 1 above)
Comment (mansona) [reply README.md:80]: ooo good catch, I didn't see this when I was reviewing 🙈

File: README.md (hunk 2 above)
Comment (vstefanovic97) [reply README.md:82]: ahhh.... yeah you are right

Comment (vstefanovic97) [reply README.md]: pushed a fix @NullVoxPopuli

**Review by NullVoxPopuli — APPROVED** [review]

**Pull request merged**